README.md

```diff
@@ -78,70 +119,15 @@ Currently, there are only two "executor" profiles that define the job execution
-
-### Using Containers
+### Location of Environments
 
-Sometimes, it is easiest to run the workflow in Docker or Singularity containers. We provide ready-made containers at [Github Container Registry](https://github.com/orgs/DKFZ-ODCF/packages).
+By default the Conda environments of the jobs as well as the Singularity containers are stored in subdirectories of the `cache/` subdirectory of the workflows installation directory (a.k.a `projectDir` by Nextflow). 
 
-#### Run with Docker
+This is suited for either a user-specific installation or for a centralized installation for which the environments should be shared for all users. Please refer to the `nextflow.config` or the `NXF_*_CACHEDIR` environment variables to change this default (see [here](https://www.nextflow.io/docs/latest/config.html#environment-variables). 
 
-You can run the workflow locally with Docker you can do e.g.
-
-```bash
-nextflow run main.nf \
-    -profile local,docker \
-    -ansi-log \
-    --input=test/test1_paired.bam,test/test1_unpaired.bam \
-    --outputDir=test_out \
-    --sortFastqs=true
-```
-
-This will automatically download the container from [Github Container Registry](https://github.com/orgs/DKFZ-ODCF/packages).
-
-#### Run with Singularity
-
-To run the workflow with [Singularity](https://singularity.lbl.gov/), convert the Docker container to Singularity:
-
-```bash
-versionTag=1.0.0
-
-cd nf-bam2fastq
-# Convert the Docker image to Singularity.
-# The singularity file is assumed to be in the workflow source directory. See nextflow.config.
-singularity \
-  build \
-  nf-bam2fastq_$version.sif \
-  docker-daemon://ghcr.io/dkfz-odcf/nf-bam2fastq:$versionTag
-
-# Run with the "singularity" profile
-nextflow run main.nf \
-    -profile local,singularity \
-    -ansi-log \
-    --input=test/test1_paired.bam,test/test1_unpaired.bam \
-    --outputDir=test_out \
-    --sortFastqs=true
-```
-
-
-## Origins
-
-The workflow is a port of the Roddy-based [https://github.com/TheRoddyWMS/BamToFastqPlugin](BamToFastqPlugin). Compared to the Roddy-workflow, some problems with the execution of parallel processing, resulting in potential errors, have been fixed.
+Make sure your users have read and execute permissions on the directories and read permissions on the files in the shared environment directories. Set `NXF_CONDA_CACHEDIR` to an absolute path to avoid "Not a conda environment:  path/to/env/nf-bam2fastq-3e98300235b5aed9f3835e00669fb59f" errors.
```

Comment:
`Not a conda environment: path`
are the doubled whitespaces correct/needed?

Reply:
Actually, this was just copy and paste from the actual error message that I got. I removed the space anyway.